scss/civicrm/dashboard/pages/_view-dashboard.scss

```diff
@@ -0,0 +1,233 @@
+.column {
+  .widget-controls {
+    padding: 17px 10px;
+
+    h3 {
+      font-family: $font-family-base;
+      font-size: $font-size-h2;
+    }
+  }
+
+  .widget-content {
+    padding: 0;
+
+    table {
+      background: transparent;
+      border: none;
+      font-family: $font-family-base;
+      margin: 7px 0;
+
+      thead {
+        border-bottom: 1px solid $crm-dark-grayblue;
+
+        th {
+          background: $gray-lighter !important;
+          border: none;
+          color: $gray-darker !important;
+          line-height: 1.5em;
+          padding: 10px 15px;
+          text-align: left;
+          vertical-align: middle;
+
+          div {
+            padding: 0;
+          }
+        }
+      }
+
+      tbody {
+
+        tr {
+          background: $crm-white;
+          border-bottom: 1px solid $crm-background;
+
+          td {
+            background: $crm-white;
+            border: none;
+            color: $gray-dark;
+            line-height: 1.5em;
+            padding: 7px 15px;
+            text-align: left;
+          }
+        }
+      }
+    }
+  }
+
+  #civicrm-news-feed {
+    .ui-widget-header {
+      border-bottom-color: $crm-dark-grayblue;
+
+      .ui-tabs-active {
+        border-bottom: 2px solid $crm-white;
+        border-color: $crm-dark-grayblue;
+        border-bottom-color: $crm-white;
+        border-radius: $border-radius-child $border-radius-child 0 0;
+      }
+
+      .ui-tabs-active a {
+        font-weight: $crm-font-weight-h3;
+      }
+
+      a {
+        font-family: $font-family-base;
+        font-weight: $crm-font-weight-h1;
+      }
+    }
+
+    [id*="civicrm-news-feed-"] {
+      padding: 0 !important;
+
+      .crm-accordion-wrapper {
```

Comment:
Is this accordion styling specific just to the news feed? Or is this how any accordions should appear throughout civicrm?

Reply:
Yes, This is how it will be displayed throughout civicrm.
See https://projects.invisionapp.com/d/main#/console/10432973/221634721/preview

Reply:
If this is how the accordion is supposed to look like everywhere in CiviCRM:

1) Shouldn't it be already styled already in shoreditch? I'm pretty sure that we already have accordion-specific css somewhere here
2) Why is your style under `#civicrm-news-feed`?

Reply:
@AkA84 The does not include all the styles, we have different style of accordion 
- Like accordion body fonts, shadow.
- Title section - font-weight, color etc.

To match the stlye, it was already done at https://github.com/civicrm/org.civicrm.shoreditch/blob/master/scss/civicrm/dashboard/_dashboard.scss
Which I now moved at dashboard/pages/_view-dashboard.scss